test/send_reminders_test.js

```diff
@@ -47,25 +47,69 @@ describe("with a reminder that hasn't been sent", function() {
                     .reply(200, {"status":200}, { 'access-control-allow-credentials': 'true'});
 
                 sendReminders().then(function(res) {
-                    knex("reminders").select("*").then(function(rows) {
+                    knex("reminders").where({ sent: true }).select("*").then(function (rows) {
 
                         console.log(JSON.stringify(rows));
 
-                        expect(rows[0].sent).to.equal(true);
+                        expect(rows.length).to.equal(1);
+
                         done();
                     }).catch(done);
                 });
 
-        }, function(err, data) {
-            if(err) {
-                console.log("ERROR", err);
+            }, function(err, data) {
+                if(err) {
+                    console.log("ERROR", err);
+                    done();
+                }
+            });
+    });
+});
+
+describe("with two reminders that haven't been sent", function () {
+    beforeEach(function (done) {
+        manager.ensureTablesExist()
+            .then(clearTable("cases"))
+            .then(clearTable("reminders"))
+            .then(loadCases([case1, case2]))
+            .then(addTestReminders([reminder1, reminder2]))
+            .then(function () {
                 done();
-            }
-        });
+            });
     });
 
+    it("sends the correct info to Twilio and updates the reminder(s) to sent", function (done) {
+        nock('https://api.twilio.com:443')
+            .filteringPath(function (path) {
+                return '/';
+            })
+            .post('/')
+            .times(2)
+            .reply(200, { "status": 200 }, { 'access-control-allow-credentials': 'true' });
+
+        knex("cases").update({ date: dates.now().add(1, 'days'), time: '02:00:00 PM', room: 'NEWROOM' })
+            .then(function () {
+                sendReminders().then(function (res) {
+                    knex("reminders").where({ sent: true }).select("*").then(function (rows) {
+
+                        console.log(JSON.stringify(rows));
+
+                        expect(rows.length).to.equal(2);
+
+                        done();
+                    }).catch(done);
+                });
+
+            }, function (err, data) {
+                if (err) {
+                    console.log("ERROR", err);
+                    done();
```

Comment:
pass the error to done here.
an easier way to write this would be to remove the second function in this call to then and put a `.catch(done)` after it.